backend/coreapp/views/scratch.py

```diff
@@ -261,9 +261,7 @@ def create_scratch(data: Dict[str, Any], allow_project: bool = False) -> Scratch
     else:
         project_function = None
 
-    libraries = [
```

Comment:
did you change this one, rather than the one I commented? :D

Reply:
oopz...